install/Makefile

```diff
@@ -17,28 +17,42 @@
 RELEASE_VER ?= ${TRAFODION_VER}-incubating
 RELEASE_TYPE ?= $(shell echo $(TRAFODION_VER_PROD)| sed -e 's/ /-/g')
 INSTALLER_TARNAME = $(shell echo ${RELEASE_TYPE}_installer-${RELEASE_VER}.tar.gz |tr '[A-Z]' '[a-z]')
+PYINSTALLER_TARNAME = $(shell echo ${RELEASE_TYPE}_pyinstaller-${RELEASE_VER}.tar.gz |tr '[A-Z]' '[a-z]')
 
-all: pkg-installer 
+all: pkg-installer
 
-pkg-installer: installer/LICENSE installer/NOTICE installer/DISCLAIMER
+pkg-installer: installer/LICENSE installer/NOTICE installer/DISCLAIMER \
+               python-installer/LICENSE python-installer/NOTICE python-installer/DISCLAIMER
 	tar czf ${INSTALLER_TARNAME} installer --exclude=tools
+	tar czf ${PYINSTALLER_TARNAME} python-installer
 	mkdir -p ../distribution
```

Comment:
Rather than build two tarballs in one makefile target, I think it would be cleaner to make another target, and have all target depend on both.